lib/locale.pm

```diff
@@ -70,7 +73,7 @@ sub import {
                               :numeric :monetary :time) );
         for (my $i = 0; $i < @_; $i++) {
```

Comment:
not related to this change but maybe consider something like this `foreach my $arg ( @_ )`